spec/seedData.test.js

```diff
@@ -0,0 +1,29 @@
+const seedDataTest = require('../data/seedDataGenerator');
+
+describe('Seed Data Generator Function Testing', () => {
+  test('randomElement should be a function', () => {
+    expect(typeof seedDataTest.randoEl).toBe('function');
```

Comment:
You should add another expect for each of these to see if they provide what you expect. For example check to see if you get a number from the function.